app/src/main/java/com/yeonae/chamelezone/view/map/MapTabFragment.kt

```diff
@@ -19,21 +17,28 @@ import com.google.android.gms.maps.CameraUpdateFactory
 import com.google.android.gms.maps.GoogleMap
 import com.google.android.gms.maps.OnMapReadyCallback
 import com.google.android.gms.maps.model.LatLng
-import com.google.android.gms.maps.model.Marker
 import com.google.android.gms.maps.model.MarkerOptions
 import com.gun0912.tedpermission.PermissionListener
 import com.gun0912.tedpermission.TedPermission
-import com.yeonae.chamelezone.SingleDialogFragment
+import com.yeonae.chamelezone.App
 import com.yeonae.chamelezone.Injection
 import com.yeonae.chamelezone.R
+import com.yeonae.chamelezone.SingleDialogFragment
 import com.yeonae.chamelezone.network.model.PlaceResponse
-import com.yeonae.chamelezone.util.App
 import com.yeonae.chamelezone.view.home.HomeActivity
 import com.yeonae.chamelezone.view.map.presenter.MapContract
 import com.yeonae.chamelezone.view.map.presenter.MapPresenter
 import kotlinx.android.synthetic.main.fragment_map_tab.*
 
 class MapTabFragment : Fragment(), OnMapReadyCallback, MapContract.View {
+    override lateinit var presenter: MapContract.Presenter
+    private lateinit var map: GoogleMap
+    private lateinit var fusedLocationProviderClient: FusedLocationProviderClient
+    private lateinit var locationRequest: LocationRequest
+    lateinit var currentLocation: Location
+    lateinit var currentLatLng: LatLng
```

Comment:
```suggestion
    lateinit var currentLocation: Location
    lateinit var currentLatLng: LatLng
```
접근한정자를 붙이지 않은 이유가 있을까요?